api_specifications/evidence-api-0.1.0.yaml

```diff
@@ -203,31 +228,70 @@ components:
       required: [tenantId, artifactId, individualId, comment]
    
     EvidenceRequest:
       type: object
       properties:
-        requestInfo:
-          $ref: >-
-            https://raw.githubusercontent.com/egovernments/DIGIT-Specs/common-contract-update/Common%20Services/common-contract.yaml#/components/schemas/RequestInfo
+        RequestInfo:
+          $ref: 'https://raw.githubusercontent.com/egovernments/DIGIT-Specs/common-contract-update/Common%20Services/common-contract.yaml#/components/schemas/RequestInfo'
         artifacts:
           type: array
           items:
             $ref: '#/components/schemas/Artifact'
     EvidenceResponse:
       type: object
       properties:
-        responseInfo:
-          $ref: >-
-            https://raw.githubusercontent.com/egovernments/DIGIT-Specs/common-contract-update/Common%20Services/common-contract.yaml#/components/schemas/ResponseInfo
+        ResponseInfo:
+          $ref: 'https://raw.githubusercontent.com/egovernments/DIGIT-Specs/common-contract-update/Common%20Services/common-contract.yaml#/components/schemas/ResponseInfo'
         artifacts:
           type: array
           items:
             $ref: '#/components/schemas/Artifact'
         pagination:
-          $ref: >-
-            https://raw.githubusercontent.com/egovernments/DIGIT-Specs/common-contract-update/Common%20Services/common-contract.yaml#/components/schemas/Pagination
-
+          $ref: 'https://raw.githubusercontent.com/egovernments/DIGIT-Specs/common-contract-update/Common%20Services/common-contract.yaml#/components/schemas/Pagination'
+    EvidenceSearchRequest:
+      type: object
+      properties:
+        RequestInfo:
+          $ref: 'https://raw.githubusercontent.com/egovernments/DIGIT-Specs/common-contract-update/Common%20Services/common-contract.yaml#/components/schemas/RequestInfo'
+        tenantid: 
+          type: string
+        criteria:
+          $ref: '#/components/schemas/EvidenceSearchCriteria'
+          
+    EvidenceSearchResponse:
+      type: object
+      properties:
+        ResponseInfo:
+          $ref: 'https://raw.githubusercontent.com/egovernments/DIGIT-Specs/common-contract-update/Common%20Services/common-contract.yaml#/components/schemas/ResponseInfo'
+        tenantid: 
+          type: string    
+        criteria:
+          $ref: '#/components/schemas/EvidenceSearchCriteria'
+          
+    EvidenceSearchCriteria:
+      type: object
+      description: the fields specified will be used in a logical AND condition
+      properties:
+        caseId:
+          type: string
+        applicationId:
+          type: string
+        hearing:
+          type: string
+        order:
+          type: string
+        sourceid:
+          type: string
+        sourcename:
+          type: string
+        status:
+          type: string
+        responseList:
```

Comment:
because the incoming criteria is not an array, the response object need not be embedded in the request object. You can very well put the responseList directly in the EvidenceSearchResponse 

    EvidenceSearchResponse:
      type: object
      properties:
        ResponseInfo:
          $ref: 'https://raw.githubusercontent.com/egovernments/DIGIT-Specs/common-contract-update/Common%20Services/common-contract.yaml#/components/schemas/ResponseInfo'  
        responseList:
          type: array
          description: contains the list of artifacts found as per the search criteria